policy/lib/image/image_test.rego

```diff
@@ -47,8 +46,15 @@ test_parse if {
 		image.parse(concat("", [repository_with_port, ":", tag, " "])),
 		{"repo": repository_with_port, "tag": tag, "digest": ""},
 	)
+}
 
-	not image.parse(concat("", [bad_repository, ":", tag, "@", digest]))
+test_not_parse if {
+	tag := "latest"
+	digest := "sha256:01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b"
+	not image.parse(concat("", ["http://not-a-registry.com", ":", tag, "@", digest]))
+	not image.parse("oci://not-a-registry.com")
```

Comment:
It might be unexpected for some that oci://valid.registry.io/org/repo fails, but it's clearly shown by this test, so we can worry about it when/if we need to. Lgtm.

Reply:
fwiw, I thought that too, but the original code seemed to block anything with `://`. https://github.com/enterprise-contract/ec-policies/blob/main/policy/lib/image/image.rego#L25